buildchain/buildchain/versions.py

```diff
@@ -360,6 +360,106 @@ def _version_prefix(version: str, prefix: str = 'v') -> str:
     ),
 )
 
-PACKAGES_MAP = {pkg.name: pkg for pkg in PACKAGES}
+PACKAGES_MAP = {pkg.name: pkg for pkg in RPM_PACKAGES}
```

Comment:
Should probably be renamed `RPM_PACKAGES_MAP`, but anyway this part will be reworked by Jordi